sklearn/neural_network/tests/test_mlp.py

```diff
@@ -334,6 +334,16 @@ def test_partial_fit_classification():
         pred2 = mlp.predict(X)
         assert_array_equal(pred1, pred2)
         assert_greater(mlp.score(X, y), 0.95)
+        
+def test_partial_fit_bug_6994():
```

Comment:
I would not use the issue number in the test name. Having the issue number in the comment is nice, but it would be good if you could add a self-contained description of the bug / test in the comment.
As name, maybe use something like ``test_partial_fit_classes`` or something.